src/software/ai/navigator/path_planner/global_path_planner.cpp

```diff
@@ -0,0 +1,59 @@
+#include "software/ai/navigator/path_planner/global_path_planner.h"
```

Comment:
one more thing I noticed: the file names should be global_path_planner_factory.h/cpp/_test.cpp

Reply:
done